single-profile.php

```diff
@@ -73,6 +73,10 @@
 							?>
 					<div class="link-list mar-right">
 							<?php
+							// Avoid undefined array index warnings.
+							$link['link']  = $link['link'] ?? '';
+							$link['title'] = $link['title'] ?? '';
```

Comment:
This patch looks good, but if you have bandwidth, could you try to test out the different profile fields using a recent Production DB dump, and see if you can figure out what data in the CMS causes this error? This patch is a bandaid, and we should know whether the data entry form for profiles is broken in some way that causes this.